operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -13,7 +13,7 @@ SCRIPTPATH="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
 source ${SCRIPTPATH}/traceUtils.sh
 [ $? -ne 0 ] && echo "Error: missing file ${SCRIPTPATH}/traceUtils.sh" && exit 1
 
-trace "Stop server ${SERVER_NAME}" &>> /weblogic-operator/stopserver.out
+trace "Stop server ${SERVER_NAME}" &>> ${STOP_OUT_FILE}
```

Comment:
STOP_OUT_FILE hasn't been defined yet, it's initialization should be moved up to line 15.

Reply:
this is fixed